### PR TITLE
(fix) alt nav tags

### PR DESCRIPTION
**Pull request by SOAP1914**

### CODE REVIEW

File: app/js/components/discovery/Sidebar.jsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ var Sidebar = React.createClass({
         return (
             <aside className="sidebar col-xs-3 col-lg-2">
                 <ul className="list-unstyled facet-group">
-                    <li className={ homeLinkClasses } id="home" onClick={ this.onHomeClick }><i className="icon-shopping"></i> Center Home</li>
+                    <li className={ homeLinkClasses } id="home" onClick={ this.onHomeClick }><i className="icon-shopping" atl=""></i> Center Home</li>
```
Comment (blaino) [inline app/js/components/discovery/Sidebar.jsx]: Says "atl" not "alt".

File: app/js/components/NavBar/index.jsx [hunk 2]
```diff
@@ -53,10 +53,10 @@ var NavBar = React.createClass({
                         <ul className="nav navbar-nav">
                             <UserNotificationDropdown/>
                             <li className="tooltiped" data-toggle="tooltip" data-placement="bottom" title="Help">
-                                <a href="#" onClick={this.showHelpModal}><i className="icon-question-grayLightest" alt="Help"></i></a>
+                                <a href="#" onClick={this.showHelpModal}><i className="icon-question-grayLightest"alt=""></i></a>
```
Comment (blaino) [inline app/js/components/NavBar/index.jsx]: might be nice to have a space before "alt"

**Comment by SOAP1914** [comment]
Thanks @blaino for the feedback will change asap

**Comment by arw180** [comment]
:+1:

**Comment by blaino** [comment]
:+1:

**Pull request merged**